app/javascript/components/public_health/CustomExport.js

```diff
@@ -406,29 +407,7 @@ class CustomExport extends React.Component {
                     onChange={event => this.handlePresetChange('name', event?.target?.value)}
                   />
                 </Col>
-                <Col md={6} className="px-1 pt-2">
-                  <Form.Group className="mb-0">
-                    <Button
-                      id="custom-export-format-csv"
-                      size="sm"
-                      variant={this.state.preset?.config?.format === 'csv' ? 'primary' : 'outline-secondary'}
-                      style={{ outline: 'none', boxShadow: 'none' }}
-                      onClick={() => this.handlePresetChange('config.format', 'csv')}>
-                      <FontAwesomeIcon className="mr-1" icon={['fas', 'file-csv']} />
-                      CSV
-                    </Button>
-                    <Button
-                      id="custom-export-format-xlsx"
-                      size="sm"
-                      variant={this.state.preset?.config?.format === 'xlsx' ? 'primary' : 'outline-secondary'}
-                      style={{ outline: 'none', boxShadow: 'none' }}
-                      onClick={() => this.handlePresetChange('config.format', 'xlsx')}>
-                      <FontAwesomeIcon className="mr-1" icon={['fas', 'file-excel']} />
-                      Excel
-                    </Button>
-                  </Form.Group>
-                </Col>
-                <Col md={6} className="px-1 pt-2">
```

Comment:
I assume from the screencaps this doesn't break the javascript but was line 431 also supposed to be removed? I don't see a corresponding /Col being cut in the diff. Aside from that, looks good to me.

Reply:
It wasn't removed, it was updated to have `md={12}`. You can see it in the diff. Also don't assume it doesn't break the javascript - please test locally as well!